### PR TITLE
INFENG-9845: Remove dart1 in CI

### PR DESCRIPTION
**Pull request by yuanmwang-wf**
### Story:
Frugal still uses dart1 in both workiva-build and skynet. Switch to dart2. 

### Acceptance Criteria:
- [ ] At least one InfRe Squad 2 member has reviewed and +1'd
- [ ] Code has been tested and results documented
- [ ] Unit tests have been updated
- [ ] Updates to documentation if necessary
- [ ] Verify and document changes to any other Messaging components
- [ ] Pull request made against the 'develop' branch, not master

### Design Notes:
TODO

### How To Test:
CI passes.

### My Test Results:
CI passed.

#### Reviewers:
@Workiva/product2

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/dart/test/transport/f_adapter_transport_test.dart [hunk 1]
```diff
@@ -30,11 +30,11 @@ void main() {
       messageStream = new StreamController.broadcast();
 
       socket = new MockSocket();
-      when(socket.onState).thenReturn(stateStream.stream);
-      when(socket.onError).thenReturn(errorStream.stream);
-      when(socket.onMessage).thenReturn(messageStream.stream);
+      when(socket.onState).thenAnswer((_) => stateStream.stream);
```
Comment (brianshannan-wf) [inline lib/dart/test/transport/f_adapter_transport_test.dart]: Why did these need changed?

File: lib/dart/test/transport/f_http_transport_test.dart [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import 'dart:async';
-import 'dart:convert' show BASE64;
+import 'dart:convert' show base64;
```
Comment (brianshannan-wf) [inline lib/dart/test/transport/f_http_transport_test.dart]: I thought the all caps version was dart1 and dart2 compatible? Maybe I'm wrong

Comment (yuanmwang-wf) [reply lib/dart/test/transport/f_http_transport_test.dart]: I'm not sure but after switching to Dart 2 it throwing errors. 

Screen Shot 2019-10-31 at 2 49 01 PM

File: lib/dart/test/transport/f_adapter_transport_test.dart (hunk 1 above)
Comment (yuanmwang-wf) [reply lib/dart/test/transport/f_adapter_transport_test.dart]: This one's throwing error after switching to Dart 2. 

Screen Shot 2019-10-31 at 2 53 33 PM

File: lib/dart/test/transport/f_http_transport_test.dart (hunk 2 above)
Comment (brianshannan-wf) [reply lib/dart/test/transport/f_http_transport_test.dart]: I think we still need to use https://github.com/Workiva/messaging-sdk/pull/674/files#diff-4c5e64b7589b045fdf24167b578b4222R10 as that's the way that supports dart1 and dart2. Notably in their [docs](https://pub.dev/documentation/dart2_constant/latest/)
```
This package is a polyfill for the core library constant names that are changing from Dart 1 to Dart 2.
```

**Comment by brianshannan-wf** [comment]
@Workiva/release-management-p

**Pull request merged**